### PR TITLE
 BCMOHAD-15804 ||Bug fix

### PR DESCRIPTION
**Pull request by ahshaik1**
Resetting both cost review and cost include checkbox values to false after first mass update

### CODE REVIEW

File: TPL/force-app/main/default/lwc/ambulanceRecordsCase/ambulanceRecordsCase.js [hunk 1]
```diff
@@ -297,10 +297,12 @@ export default class AmbulanceRecordsCase extends LightningElement {
     }
     changeCostReview(event){
         this.costReview = event.target.checked;
+        console.log(this.costReview);
         
     }
     changeCostInclude(event){
```
Comment (cgijeffolsen) [inline TPL/force-app/main/default/lwc/ambulanceRecordsCase/ambulanceRecordsCase.js:303]: Important cosmetic change -- to not keep the form values after submission.

**Review by cgijeffolsen — APPROVED** [review]
@ahshaik1 Approved, thank you

**Review by NataliaNikishina — APPROVED** [review]
@ahshaik1 approved
cc: @rahulmantriSFDC

**Review by rahul-mantri — APPROVED** [review]
Approved @ahshaik1 

CC: @NataliaNikishina

**Pull request merged**